security/pfSense-pkg-tinc/files/usr/local/www/status_tinc.php

```diff
@@ -26,8 +26,15 @@ function tinc_status_usr1() {
 	usleep(500000);
 	$clog_path = "/usr/local/sbin/clog";
 	$result = array();
+	$logfile = "/var/log/tinc.log";
+	$firmware = host_firmware_version();
 
-	exec("{$clog_path} /var/log/tinc.log | /usr/bin/sed -e 's/.*tinc\[.*\]: //'", $result);
+	if ($firmware['config_version'] >= 19.7) {
```

Comment:
This isn't a proper version test. The config revision is not a reliable indicator of the pfSense version. It should be based on `$g['product_version']` or similar to test the actual version directly.

Reply:
No, it is not. That does not necessarily determine the pfSense version either. It must use product_version. If you need to compare, there are convenience functions like `pfs_version_compare()` that facilitate a proper comparison.